src/themes/greyVest/SearchFilters.js

```diff
@@ -19,7 +19,20 @@ let LabelledList = ({ list, Component }) =>
     list
   )
 
-let BasicSearchFilters = ({ setMode, trees, children, FiltersBox }) => (
+export let AddableFilterList = props => (
+  <>
+    <FilterList {...props} />
+    <FilterAdder {...props} uniqueFields />
+  </>
+)
+
+export let FiltersBox = props => (
```

Comment:
If we move this on the theme (and potentially unify with AddableFilterList), everything else in here seems like it could move to layout